com/madishetti/JedisManager.cfc

```diff
@@ -95,11 +95,17 @@ component accessors="true" {
         try {
             // Get a Jedis resource from the pool
             jedis = getJedisResource();
+
+            var cacheData = arguments.dataToCache;
+            // Check if the data to cache is a non-simple value
+            if (!isSimpleValue(cacheData)) {
```

Comment:
One thing to check here - where are some things, like - say - CFC instances, that would probably to into the if block and then try to serialise. Is there an opportunity here to explicitly check for the types you want to allow being serialised. Maybe Struct, Array, Query, XML? 

Just an idea, might need some thought.